client/src/components/content/ModelerSidebar/PropertiesPanel/PropertiesPanelComponents/PPParameterInput.jsx

```diff
@@ -39,16 +42,18 @@ const PPParameterInput = ({
   );
 
   /**
-   * @description changes the state for "userInputRequired" and also the parameter value 
+   * @description changes the state for "userInputRequired" and also the parameter value
    * @param {String} parameterName Name of the currently handled parameter
    */
   const changeUserInputRequirement = (parameterName) => {
+    setParameterValue('');
     setPropertyForParameter(
       selectedActivity,
       parameterName,
       'requireUserInput',
       !userInputRequired
     );
+    setPropertyForParameter(selectedActivity, parameterName, 'value', '');
```

Comment:
Why is this needed as well as line 49? Just for my understanding :)

Reply:
With these lines we delete the currently stored value of the input fields in the sidebar. These values are written to the Value field of the Parameter object, which must be empty for the SSOT => ROBOT parser to work correctly.